wrapper/pre_copy.py

```diff
@@ -340,6 +349,12 @@ def __init__(self, nbd, disk, tmp_dir):
         self.proc_qemu = None
         self.overlay = None
 
+        datastore = disk.backing.datastore
+        datastore_mountpoint = datastore.summary.url.replace("ds://", "")
+        self.absolute_path = "%s/%s" % (
+                datastore_mountpoint,
+                disk.backing.fileName.replace("[%s] " % datastore.name, ""))
```

Comment:
Something like:
```
fileName = disk.backing.fileName
ds_id = "[%s] " % datastore.name
if fileName.startswith(ds_id):
    fileName = fileName[len(ds_id):]
```
or even, if you are sure both strings will always start with the proper prefix, something just this:
```
datastore.summary.url[len("ds://"):] + '/' + disk.backing.fileName[len('[%s] ' % datastore.name):]
```
would be safer, just in case there is some mr. smartypants who has `[datastore] ` or as part of the filename.